staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -400,6 +400,33 @@ type ListOptions struct {
 	// This field is not supported when watch is true. Clients may start a watch from the last
 	// resourceVersion value returned by the server and not miss any modifications.
 	Continue string `json:"continue,omitempty" protobuf:"bytes,8,opt,name=continue"`
+
+	// SendInitialEvents, when set together with Watch option,
+	// begin the watch stream with synthetic init events to build the
+	// whole state of all resources followed by a synthetic "Bookmark"
+	// event containing a ResourceVersion after which the server
+	// continues streaming events.
+	//
+	// When SendInitialEvents option is set, we require ResourceVersionMatch
+	// option to also be set. The semantic of the watch request is as following:
+	// - ResourceVersionMatch = NotOlderThan
```

Comment:
```suggestion
	// - `resourceVersionMatch` = NotOlderThan
```

(multiple instances below)